### DISCUSSION TITLE
How should I play the add-on?

### DISCUSSION THREAD

**Discussion by taccer** [hassio-addons/addon-spotify-connect#88] [Q&A] [answered]
Hi! 
I'm kind of a starter with HA. I manage some things...

After install and I have set user/pw...the addon starts w/t no error.
But I have no insight how to "play" in the addon.
Sure, I have read some of discussions here and community-ha..

Ok, there is no entity, mediaplayer...so how do I start a song and play it?

Running HA at Ubuntu 20.04 on a PC in a vbox(6+)
 System: Home Assistant OS 5.12  (amd64 / qemux86-64)
 Home Assistant Core: 2021.3.1
 Home Assistant Supervisor: 2021.03.4

**Comment by frenck** [comment, marked as answer]
It is a Spotify Connect receiver... Play on it from your Spotify account / existing players. It does not provide an interface (as it is a receiver)

**Reply by taccer** [reply]
Ah, I figured as much...
I'll will just have to find a player in HA, huh.
Ok, will do.
Tnx
